### DISCUSSION TITLE
ShakeMarks and Bookmark Export

### DISCUSSION THREAD

**Discussion by hacklavya** [PaulWoitaschek/Voice#1313] [Ideas]
I am using it from around a year, I want following features:-
# 1. Shakemark: 
While listening on the bike I want to bookmark some location on audio file, but for that  I have to stop  my bike and take out the phone and bookmark location, with shakemark feature I can shake my phone in a predefined manner to bookmark that playback position.

# 2. import-export: 
This is just to import-export the bookmarks in case I want to reset my phone.

**Comment by versavius** [comment]
What about using an app like Macrodroid or Tasker to automate the bookmarking when shaking the device? I do that (shake the device) and macrodroid turns the light on and off.

I haven't tried with what your suggesting of shaking to do a bookmark but it might be worth you trying that.

**Comment by hacklavya** [comment]
Thanks for the suggestion, I was not aware of such external apps.
I will try to use these.
